build.sbt

```diff
@@ -1197,13 +1225,9 @@ lazy val runtime = (project in file("engine/runtime"))
       "com.novocode"        % "junit-interface"       % "0.11"            % Test exclude ("junit", "junit-dep")
     ),
     // Note [Unmanaged Classpath]
-    Test / unmanagedClasspath += (baseDirectory.value / ".." / ".." / "app" / "gui" / "view" / "graph-editor" / "src" / "builtin" / "visualization" / "native" / "inc"),
```

Comment:
If you move this line, I think you should also move the associated comment.

Reply:
Also btw. do you know why is that there?

Reply:
Yeah, one of tests depends on the file shared between gui and engine

Reply:
I see, I still don't understand why we need to signalize this dependency. The test will just load this file at runtime, right? So I don't know why SBT must know about it (I'm not saying it doesn't have to, I just cannot see the reason).

Reply:
how else would it know without hardcoding the path?

Reply:
Oh, no way around it.

My question is - why does it have to know?

Reply:
Because of this single test:
```
[info] - should run internal IDE visualisation preprocessor catching error *** FAILED ***
[info]   java.io.FileNotFoundException: resource 'error_preprocessor.enso' was not found in the classpath from the given classloader.
[info]   at scala.io.Source$.fromResource(Source.scala:183)
```